src/transform_nodelet_ros2.cpp

```diff
@@ -10,7 +10,7 @@ namespace transform_nodelet_ns {
 
 // Creating constructor
 OvtransformNodeletClass::OvtransformNodeletClass(const rclcpp::NodeOptions& transform_node)
-    : Node("odom_transform_nodelet", transform_node) {
+    : Node("odom_transform_nodelet", transform_node), mTfBr(this) {
```

Comment:
why do we need to have this as the argument?

Reply:
To initialize the transform broadcaster. Otherwise it complains.